src/shared/contexts/editor/insertFunction.ts

```diff
@@ -24,11 +18,11 @@ export const generateImport = (
   }
 
   if (
-    !funcPackage ||
-    funcPackage === 'universe' ||
+    !func.package ||
+    func.package === 'universe' ||
     script.includes(importStatement)
   ) {
-    return false
```

Comment:
Why the change here?

Reply:
This method is now combining two separate (but functionally equivalent) ways to generate the import statement. One approach produced a string or null, whereas the other produced a string or false. 

It's cleaner to use the former option, because typescript accepts a returned null as unifying with a string. (Versus having to provide a `string | boolean` type.) And we do a null-pointer check downstream anyways.